bootstrap.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+curl -L https://github.com/jolly-good-toolbelt/jgt_tools/raw/master/env_setup.py | python
```

Comment:
Should this be `python3` since we're only running on Python 3? I know @rbcasperson has had problems in the past with not being explicit about which version of Python.

Reply:
Good catch.